blazor/pivot-table/events.md

```diff
@@ -495,25 +495,20 @@ The event [ToolbarRender](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazo
                 <PivotViewFormatSetting Name="Amount" Format="C"></PivotViewFormatSetting>
             </PivotViewFormatSettings>
         </PivotViewDataSourceSettings>
-        <PivotViewEvents TValue="ProductDetails" ToolbarRendered="toolbarRender" RenameReport="renameReport" RemoveReport="removeReport" SaveReport="saveReport" LoadReport="loadReport" FetchReport="fetchReport" ></PivotViewEvents>
+        <PivotViewEvents TValue="ProductDetails" ToolbarRendered="ToolbarRender" ></PivotViewEvents>
         <PivotViewGridSettings ColumnWidth="140"></PivotViewGridSettings>
 </SfPivotView>
 
-@code{
-    public List<ToolbarItems> toolbar = new List<ToolbarItems> {
-        ToolbarItems.New,
-        ToolbarItems.Load,
-        ToolbarItems.Remove,
-        ToolbarItems.Rename,
-        ToolbarItems.SaveAs,
-        ToolbarItems.Save,
-        ToolbarItems.Grid,
-        ToolbarItems.Chart,
-        ToolbarItems.Export,
-        ToolbarItems.SubTotal,
-        ToolbarItems.GrandTotal,
-        ToolbarItems.ConditionalFormatting,
-        ToolbarItems.FieldList
+@code {
+    public List< Syncfusion.Blazor.PivotView.ToolbarItems> toolbar = new List<Syncfusion.Blazor.PivotView.ToolbarItems> {
```

Comment:
public declaration should be Pascal case. Eg:- `Toolbar`. If public declaration required mean, make it as private.

Reply:
resolved